sound/pci/hda/hda_intel.c

```diff
@@ -513,7 +513,7 @@ static void intel_init_lctl(struct azx *chip)
 
 	/* 0. check lctl register value is correct or not */
 	val = readl(bus->mlcap + AZX_ML_BASE + AZX_REG_ML_LCTL);
-	/* if SCF is already set, let's use it */
+	/* only perform additional configurations if the SCF is initially based on 6MHz */
```

Comment:
according to original patch 6MHz is a hw default reset value ?  It is a little hard to get it why talk about 6MHz without any context

Reply:
No the hardware default has always been 0x2: 24 MHz for obvious backwards-compatibility reasons. The BIOS might select a different setting.